lib/topic_view.rb

```diff
@@ -713,6 +713,10 @@ def load_mentioned_users
     @mentioned_users = mentioned_users.to_h { |u| [u.username, u] }
   end
 
+  def tags
+    @topic.tags.map(&:name)
```

Comment:
Are `tags` properly (pre-)loaded to ensure there won't be an N+1?

Reply:
Just checked and indeed they weren’t eagerly loaded :+1: